tests/apps/test_cost.py

```diff
@@ -79,9 +155,12 @@ def test_h_reparametrized(self, vgbs, dim, params):
         probs_init = [vgbs.prob_sample(np.zeros(dim - 1), s) for s in possible_samples]
 
         cost = sum([probs[i] * h(s) for i, s in enumerate(possible_samples)])
-        cost_reparam = sum([probs_init[i] * cost_fn.h_reparametrized(s, params) for i, \
-                                                                                    s in enumerate(
-            possible_samples)])
+        cost_reparam = sum(
+            [
+                probs_init[i] * cost_fn.h_reparametrized(s, params)
+                for i, s in enumerate(possible_samples)
+            ]
+        )
```

Comment:
Too many lines?

Reply:
I've tried to simplify by defining the terms inside the sum on a previous line. However, the above is how it looked when running `black -l 100` so I wouldn't worry too much about second guessing the formatting.